public/whoweare.html

```diff
@@ -37,19 +37,21 @@ <h1>#findthemasks</h1>
     Catherine Chang<br>
     Melanie Cohen<br>
     Melissa Kerr<br>
-    <a href="http://twitter.com/sean_m_kelleher">Seán Kelleher</a><br>
+    <a href="http://twitter.com/sean_m_kelleher">SeÃ¡n Kelleher</a><br>
```

Comment:
It's interesting this came out this way in Github, it looks like this in my local dev (unaffected). I think maybe I'll use `&aacute;` here to be safe since this looks like an old fashioned ISO-8859-1 encoding/transliteration bug (even though everything's defined as UTF-8). 🤔

![2020-03-23_01-58-39](https://user-images.githubusercontent.com/4269377/77299680-10c7ef80-6caa-11ea-98b0-93a17bb66a2f.png)